src/sender-ui.py

```diff
@@ -21,70 +24,75 @@ def poll():
         sender.get_streams()
 
 
-def send_file():
-    global continue_sending
-    continue_sending = True
-    while continue_sending:
-        if sender.pending_streams:
-            print(
-                f"Pending Streams: {sender.pending_streams}, Completed Streams: {sender.completed_streams}"
-            )
-            stream_id = sender.pending_streams.pop(0)
-            ip, port = sender.consume_stream(stream_id)
-            if ip and port:
-                # To give time for receiver to start
-                # Need to find a more elegant solution in the future
-                time.sleep(3)
-                subprocess.Popen(
-                    [
-                        "ffmpeg",
-                        "-re",
-                        "-i",
-                        choose_file_entry.get(),
-                        "-f",
-                        "mpegts",
-                        "-v",
-                        "warning",
-                        f"srt://{ip}:{port}?pkt_size=1316",
-                    ]
-                )
-            sender.completed_streams.append(stream_id)
-
-
-def send_cam():
+def send(use_webcam: bool):
     global continue_sending
     continue_sending = True
     with open("config.json") as json_config:
         config = json.load(json_config)
     webcam = config["camera"]["name"]
     while continue_sending:
         if sender.pending_streams:
-            print(
-                f"Pending Streams: {sender.pending_streams}, Completed Streams: {sender.completed_streams}"
-            )
+            print(f"Pending Streams: {sender.pending_streams}, Completed Streams: {sender.completed_streams}")
+
             stream_id = sender.pending_streams.pop(0)
-            ip, port = sender.consume_stream(stream_id)
+            ip, port, is_rendezvous = sender.consume_stream(stream_id)
             if ip and port:
                 # To give time for receiver to start
                 # Need to find a more elegant solution in the future
                 time.sleep(3)
+                if use_webcam:
+                    start_ffmpeg_webcam(webcam)
+                else:
+                    start_ffmpeg_file()
+
+                if is_rendezvous:
+                    srt_url = f"{SRT_SCHEME}://{ip}:{port}?mode=rendezvous"
+                else:
+                    srt_url = f"{SRT_SCHEME}://{ip}:{port}"
+
                 subprocess.Popen(
                     [
-                        "ffmpeg",
-                        "-f",
-                        "dshow",
-                        "-i",
-                        f"video={webcam}",
-                        "-f",
-                        "mpegts",
-                        "-v",
-                        "warning",
-                        f"srt://{ip}:{port}?pkt_size=1316",
+                        "srt-live-transmit",
```

Comment:
I figured we probably lost the first few microseconds of the stream, so I thought it wasn't a big deal. But since I refactored that code, I added those changes.